free-programming-cheatsheets.md

```diff
@@ -9,7 +9,7 @@
 * [Markdown](#markdown)
 * [MongoDB](#mongodb)
 * [Python](#python)
-
+* [Kubernetes](#Kubernetes)
```

Comment:
This Kubernetes in the Index section needs to be alphabetical, so it should be after JavaScript but before Markdown at the time of this review. After doing so in the Index section, it should also be reflected below.